ops/gulp/dev-watch.js

```diff
@@ -21,6 +21,9 @@ const task = () => {
 	//update styleguide styles
 	gulp.watch([`./docs/styleguide/styles/*.scss`], ['docs-styleguide']);
 
+	//watch docs js
+	gulp.watch([`./src/scripts/*.js`,`./ops/gulp/*.js`], ['docs-js']);
```

Comment:
@xak deleted ./ops/gulp/*.js from docs-js and added watcher for JS-files in dev-serve.js